video/clip.go

```diff
@@ -5,10 +5,13 @@ import (
 	"github.com/grafov/m3u8"
 	"github.com/livepeer/catalyst-api/log"
 	ffmpeg "github.com/u2takey/ffmpeg-go"
-	"net/url"
 	"time"
 )
 
+const (
+	ClipStorageDir = "/tmp/clip_stage"
```

Comment:
I think we should use golang's built in tmp functions e.g. [os.MkdirTemp()](https://pkg.go.dev/os#MkdirTemp)

https://github.com/livepeer/catalyst-api/blob/506e24fe359d53362d154d37c72541d7876baff7/pipeline/ffmpeg.go#L279

Reply:
Sure, done.